CHANGELOG.md

```diff
@@ -39,6 +41,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Routine `print_totals` now prints small and/or large numbers in scientific notation
 - Truncate names in benchmark & emissions tables to improve readability
 - Add TransportTracers species names to `gcpy/emissions_*.yml` files
+- Folder `gcpy/gcpy` is now  `gcpy/src/gcpy`, for adherence to Python packaging standards
```

Comment:
Is this still the case or are we not using the `src` directory?

Reply:
Nice catch @msulprizio.  I'll edit the changelog.  If we use `gcpy/src` then we'd have to add `path/to/gcpy/src` to `PYTHONPATH`, which is a little ugly.

Reply:
Now removed in commit cf6fc12